R/descriptives_helper.R

```diff
@@ -211,42 +220,42 @@ variableCols <- sub.inputForDescriptives[which(sub.inputForDescriptives[,"type"]
 
 
 ### kennwerte.skala.fake(dat=dat, variableCols = c("Semz19_a", "Semz19_b", "Semz19_c", "Semz19_d"), missingValues = c(-98,-99))
-kennwerte.skala.fake <- function(dat,variableCols, missingValues = NULL) {
+#kennwerte.skala.fake <- function(dat,variableCols, missingValues = NULL) {

```

Comment:
LGTM
I would suggest to remove this before merging into main.

Reply:
is removed ...